design/recovery-internals.md

```diff
@@ -118,7 +118,7 @@ Consider an old generation with three TLogs: `A, B, C`. Their durable versions a
 * Situation 2: A tLog may die after it reports alive to the CC in the RECRUITING phase. This may cause the `knownCommittedVersion` calculated by the CC in this phase to no longer be valid in the next phases. When this happens, the CC will detect it, terminate the current recovery, and start a new recovery.
 
 
-Once we have a `knownCommittedVersion`, the CC will reconstruct the transaction state store (txnStateStore) by peeking the txnStateTag in oldLogSystem.
+Once we have a `knownCommittedVersion`, the CC will reconstruct the transaction state store (txnStateStore, https://github.com/apple/foundationdb/blob/master/design/transaction-state-store.md) by peeking the txnStateTag in oldLogSystem.
```

Comment:
Nit: Markdown can link doc this way:
```
[transaction state store](https://github.com/apple/foundationdb/blob/master/design/transaction-state-store.md)
```
which will generate hyper links by a viewer.

Reply:
Good to know. Updated. 
Thanks